onnxruntime/python/tools/transformers/models/stable_diffusion/demo_utils.py

```diff
@@ -246,6 +246,8 @@ def parse_arguments(is_xl: bool, parser):
 
     group = parser.add_argument_group("Options for ORT_CUDA engine only")
     group.add_argument("--enable-vae-slicing", action="store_true", help="True will feed only one image to VAE once.")
+    group.add_argument("--max-cuda-graphs", type=int, default=1, help="Max number of cuda graphs to use. Default 1.")
```

Comment:
shall we mention to use ORT nightly here?

Reply:
No needed. The readme mentioned that a step to build from source:
https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/python/tools/transformers/models/stable_diffusion/README.md#build-onnxruntime-from-source